features/support/env.rb

```diff
@@ -42,6 +42,15 @@
   log.error(message + "failed")  if scenario.failed?
 end
 
+After('@users') do
+  user_name = config["features"]["users"]["user_name"]
+  users = control_node.openstack.user.list
+  user_names = users.map(&:name)
+  if user_names.include? user_name
+    control_node.openstack.user.delete user_name
+  end
+end
+
```

Comment:
Please don't use tagged hooks. We are mixing here too specific implementation details from the `users` feature with high level code. 
This could be done more transparently in `Background` with `And the test user does not exists" ` right after the first `Given`.